core-bundle/src/Resources/contao/controllers/BackendFile.php

```diff
@@ -135,7 +135,7 @@ public function run()
 		}
 
 		/** @var FileSelector $strClass */
-		$strClass = $GLOBALS['BE_FFL']['fileSelector'];
+		$strClass = $GLOBALS['BE_FFL']['fileSelector'] ?? null;
```

Comment:
If this is anything but a valid class string `new $strClass()` will fail with a fatal error. What about throwing an exception in this case? 

```suggestion
		if(!$strClass = ($GLOBALS['BE_FFL']['fileSelector'] ?? false)) {
			throw new \RuntimeException('A FileSelector class must be defined.');
		}
```